_docker/environments/awestruct-production/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '2'
+services:
+  awestruct:
+    build:
+     context: ../../awestruct
+    command:
+      - "rake git_setup clean gen[drupal_production]"
+    volumes:
+      - ../../../:/home/awestruct/developer.redhat.com
+      #- ../../awestruct/overlay/ssh-key:/home/awestruct/.ssh
+    environment:
+      - google_api_key
+      - dcp_user
+      - dcp_password
+      - vimeo_client_secret
+      - vimeo_access_token_secret
+      - vimeo_access_token
+      - cache_password
+      - cache_url
+      - site_base_path
+      - site_path_suffix
+      - cdn_prefix
+      - cache_user
+      - github_token
+      - drupal_user
+      - drupal_password
+  #
+  # Testing
+  #
+  unit_tests:
+    build:
+      context: ../../awestruct
+    volumes:
+      - ../../../:/home/awestruct/developer.redhat.com
+    entrypoint: "bundle exec rake test"
+
```

Comment:
Rather than creating another environment, could we just use the existing drupal-production one and simply remove the `links:` directive from the `awestruct` service definition in the `drupal-production/docker-compose.yml`?